src/api/app/components/token_card_component.html.haml

```diff
@@ -0,0 +1,24 @@
+.card.m-1.p-2
+  .card-body
+    %h5.card-title
```

Comment:
If a token doesn't have a name, I would display _No name_ (in italic) instead of an empty `<h5>`.

Reply:
Sounds good to me, adapted